controller/userWebController.js

```diff
@@ -1946,6 +1946,67 @@ module.exports = {
                 db_connection.release();
             }
         }
+    ], 
+
+    editManagerProfileById: [
+        /*
+        JSON
+        {
+            "managerId" : "<managerId>" INTEGER,
+            "managerEmail" : "<managerEmail>",
+            "managerPassword" : "<managerPassword>",
+            "managerName" : "<managerName>",
+            "managerRole" : "<0/1>",
+            "accountStatus" : "<0/1/2>"
+        }
+        */ 
+        webTokenValidator,
+        async (req, res) => {
+            if (req.body.userRole === null || req.body.userRole === undefined || req.body.userRole === "" || (req.body.userRole !== "1" && req.body.userRole !== "0" && req.body.userRole !== "2") ||
```

Comment:
this is for the manager to update by themselves. hence userRole can only be `0`.